README.md

```diff
@@ -423,6 +423,28 @@ We strictly respect this injection hierarchy:
 | `UseCase` | `Store` |
 | `Store` | `Dao`, `Persistence`, `ApiService` |
 
+## SavedStateHandle
```

Comment:
Mohol by si to nahodiť aj do example modulu nech je vidieť reálne použitie

Reply:
Se obávám že reálné použití v našich examplech asi ani nejde...